gr-utils/python/utils/plot_fft_base.py

```diff
@@ -88,7 +87,7 @@ def get_data(self):
 
     def dofft(self, iq):
         N = len(iq)
-        iq_fft = numpy.fft.fftshift(fftpack.fft(iq))       # fft and shift axis
+        iq_fft = numpy.fft.fftshift(fft_detail.fft(iq))       # fft and shift axis
```

Comment:
What about numpy.fft?

Reply:
Well, this could be numpy.fft depending on your numpy version.

Reply:
> I guess the comment message should be adjusted as well, it does not fail silently this time.

good point!